platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLightLaf.properties

```diff
@@ -67,6 +67,10 @@ SlidingButton.attentionForeground=$ViewTab.attentionForeground
 PropSheet.setBackground=darken(@background,10%)
 PropSheet.setForeground=@foreground
 
+# Start Page
+nb.startpage.topbar.background=darken($TabbedPane.focusColor, 25%)
+nb.startpage.tab.border1.color=darken($TabbedPane.focusColor, 25%)
+nb.startpage.tab.border2.color=darken($TabbedPane.focusColor, 25%)
```

Comment:
Better use this to give tab borders same color as background:
~~~
nb.startpage.tab.border1.color=$nb.startpage.topbar.background
nb.startpage.tab.border2.color=$nb.startpage.topbar.background
~~~

Reply:
Thank you for your suggestion! I completely agree.